pkgs/development/tools/devcontainers-cli/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  nodejs_18
+}:
+let
+  pname = "devcontainers-cli";
+  version = "0.54.1";
+  hash = "sha256-L6sVDmKuFmrf9Bm9M54ABmmWFVB8ZlqU+5gAeITBS1Q=";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchzip {
+    inherit hash;
+    url = "https://registry.npmjs.org/@devcontainers/cli/-/cli-${version}.tgz";
+  };
+
+  dontBuild = true;
+  dontConfigure = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -a "$src/." "$out"
+    rm devcontainer.js
```

Comment:
Forgive me if I'm reading this wrong, but doesn't this just remove `devcontainer.js` from the build directory?